README.md

```diff
@@ -55,9 +52,20 @@ To scan your project for dependency issues, run:
 deptry .
 ```
 
-_deptry_ can be configured by using additional command line arguments, or by adding a `[tool.deptry]` section in _pyproject.toml_.
+Example output could look as follows:
+
+```terminal
+Scanning 1 files...
```

Comment:
```suggestion
Scanning 2 files...
```
Feels weird to see plural with 1 file (we might want to update the code to handle that in another PR).